### DISCUSSION TITLE
Prepare to release Apache RocketMQ 4.9.5

### DISCUSSION THREAD

**Discussion by RongtongJin** [apache/rocketmq#6388] [General]
Hello RocketMQ Community,

This is the discussion for the release of Apache RocketMQ 4.9.5 Release.

If there are any important pull requests that need to be merged into the 4.9.x LTS branch, please let me know.

**Comment by galaxy077** [comment]
Whether the client - side flow control can also come in?

**Reply by RongtongJin** [reply]
Which issue or pr are you talking about？
